src/js/actions.js

```diff
@@ -235,7 +234,8 @@ export function uploadFile(file, uiOptions, onProgress, onChange, onError) {
       } else {
         let errorMessage = req.statusText;
         if (req.status === 429) {
-          errorMessage = `You’ve reached the limit for the number of submissions we can accept at this time. Please try again in ${timeFromNow(moment.unix(parseInt(req.getResponseHeader('x-ratelimit-reset'), 10)))}.`;
+          const resetDate = new Date(req.getResponseHeader('x-ratelimit-reset') * 1000);
```

Comment:
Noting that the date conversion here looks right since the original was in seconds-since-epoch and JS is milliseconds-since-epoch. There [seems to be some controversy](https://github.com/zalando/restful-api-guidelines/issues/379) about whether this is a *relative* time or an *absolute* one. I can see the point about absolute being risky because it assumes relatively synchronized clocks. In any case, this still does whatever it did before, whether right or wrong. 😺